src/js/bootstrap-slider.js

```diff
@@ -1148,6 +1148,7 @@ const windowIsDefined = (typeof window === "object");
 
 				this.handle1.style[this.stylePos] = positionPercentages[0]+'%';
 				this.handle1.setAttribute('aria-valuenow', this._state.value[0]);
+				this.handle1.setAttribute('aria-valuetext', this.options.formatter(this._state.value[0]));
```

Comment:
Can you add a unit test that checks if the `aria-valuetext` is updated appropriately when the slider value is changed? Thanks!